build.gradle

```diff
@@ -13,7 +13,7 @@ import java.nio.file.Files
 import org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestFramework
 
 plugins {
-    id 'java'
+    id 'java-library'
```

Comment:
Any specific reason to change the plugin here?

Reply:
> Any specific reason to change the plugin here?

api() doesn't work if java-library not imported.

Reply:
`java` is actually deprecated, `java-library` is the new thing if you have a project intended to be consumed by other projects (which the SDK does).